libespm/include/libespm/CTDA.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+namespace espm {
+
+struct CTDA
+{
+  enum OperatorFlags : uint8_t
```

Comment:
I think it's better to use scoped enums. Here's cppref [link](https://en.cppreference.com/w/cpp/language/enum#:~:text=linkage%20purposes.-,Scoped%20enumerations,-enum%20struct%7Cclass) you are not familiar with them.

Reply:
Thanks for your suggestion! Implemented.